single_kernel_mongo/managers/mongo.py

```diff
@@ -544,13 +544,13 @@ def get_statuses(self, scope: Scope, recompute: bool = False) -> list[StatusObje
             # AutoReconnect is raised when a connection to the database is lost and an attempt to
             # auto-reconnect will be made by pymongo.
             logger.debug("Got error: %s, while checking replica set status", str(e))
-            return [MongodStatuses.WAITING_RECONNECT.value]
+            return [MongodStatuses.WAITING_RECONNECTION.value]
         except OperationFailure as e:
             logger.warning("Authentication Failed: %s", e, exc_info=True)
             return [MongodStatuses.WAITING_RECONFIG.value]
         except MissingCredentialsError as e:
             logger.warning("Missing credentials: %s", e, exc_info=True)
-            return [MongodStatuses.MISSING_CREDENTIALS]
+            return [MongodStatuses.WAITING_CREDENTIALS]
```

Comment:
Wooopsie I think we had forgotten to return the value in this case.
```suggestion
            return [MongodStatuses.WAITING_CREDENTIALS.value]
```